v2/metadata/client_test.go

```diff
@@ -84,9 +84,9 @@ func Test_GetPipeline(t *testing.T) {
 	mlmdClient, err := NewTestMlmdClient()
 	fatalIf(err)
 
-	_, err = client.GetPipeline(ctx, "get-pipeline-test", runId)
+	_, err = client.GetPipeline(ctx, "get-pipeline-test", runId, "kubeflow", "workflows.argoproj.io/hello-world-abcd")
```

Comment:
Do we need to write "kubeflow" and "workflows.argoproj.io" as constants?

Reply:
My gut feeling is we don't need to, because these values can be arbitrary in tests.

Reply:
rethinking over this, I think you are right, if I set up default values, they help reduce information noise in each test.